pkg/crypto/clef/clef.go

```diff
@@ -109,3 +118,14 @@ func (c *clefSigner) SignTx(transaction *types.Transaction) (*types.Transaction,
 func (c *clefSigner) EthereumAddress() (common.Address, error) {
 	return c.account.Address, nil
 }
+
+// SignTypedData signs data according to eip712
```

Comment:
period at end of sentence (goes for the rest of this file)

Reply:
fixed